branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/readwrite/IBranchLayoutWriter.java

```diff
@@ -1,12 +1,14 @@
 package com.virtuslab.branchlayout.api.readwrite;
 
-import java.nio.file.Path;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import io.vavr.collection.List;
 
 import com.virtuslab.branchlayout.api.BranchLayout;
-import com.virtuslab.branchlayout.api.BranchLayoutException;
 import com.virtuslab.qual.guieffect.UIThreadUnsafe;
 
 public interface IBranchLayoutWriter {
   @UIThreadUnsafe
```

Comment:
Actually, let's ditch that `@UIThreadUnsafe`, now that we write to VFS on UI thread anyway :trollface: